Assets/Scripts/RandomItemDrop.cs

```diff
@@ -16,7 +16,7 @@ private void Start()
         potHealth = GetComponent<NonPlayerHealth>();
     }
 
-    private void Update()
+    public override void Death()
     {
         // Zero health makes it "break".
         if (potHealth.Health <= 0)

```

Comment:
Do we still need to check for health being zero if NonPlayerHealth is handling this call?